src/NuGet.Clients/NuGet.PackageManagement.UI/Resources.resx

```diff
@@ -844,4 +844,22 @@ Please see https://aka.ms/troubleshoot_nuget_cache for more help.</value>
   <data name="Accessibility_PackageDetailsUrlLink" xml:space="preserve">
     <value>Package details link</value>
   </data>
+  <data name="Label_Deprecated" xml:space="preserve">
+    <value>Deprecated</value>
+  </data>
+  <data name="Label_OrAbove" xml:space="preserve">
+    <value> (or above)</value>
+  </data>
+  <data name="Label_DeprecationReasons_CriticalBugs" xml:space="preserve">
+    <value>This package has been deprecated because it has critical bugs.</value>
```

Comment:
What do you think of this?

```suggestion
    <value>This package has been deprecated due to critical bugs.</value>
```

Reply:
Can we have same sentence construct: "This package has been deprecated as it has critical bugs"?

Reply:
Same here. What about gallery? The unmodified text is currently shipping as part of gallery. Any change we make here ideally also applies to the gallery so we have consistent messages. https://github.com/NuGet/NuGetGallery/blob/master/src/NuGetGallery/Views/Packages/_DisplayPackageDeprecation.cshtml